src/qt/forms/optionsdialog.ui

```diff
@@ -226,6 +226,13 @@
             </property>
            </widget>
           </item>
+          <item>
+           <widget class="QCheckBox" name="m_enable_create_unsigned">
+            <property name="text">
+             <string>Enable Create Unsigned button</string>
```

Comment:
Maybe this should be a more general "Enable Advanced PSBT Controls"?

Reply:
Done